TelAviv/TAfait006.xml

```diff
@@ -0,0 +1,125 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<?xml-model href="https://raw.githubusercontent.com/BetaMasaheft/Schema/master/tei-betamesaheft.rng" schematypens="http://relaxng.org/ns/structure/1.0"?>
+<?xml-model href="https://raw.githubusercontent.com/BetaMasaheft/Schema/master/tei-betamesaheft.rng" type="application/xml" schematypens="http://purl.oclc.org/dsdl/schematron"?>
+<TEI xmlns="http://www.tei-c.org/ns/1.0" type="mss" xml:lang="en" xml:id="TAfait006">
+    <teiHeader xml:base="https://betamasaheft.eu/">
+        <fileDesc>
+            <titleStmt>
+                <title>Faitlovich 6: Falasha prayers</title>
+            </titleStmt>
+            <editionStmt>
+                <p>The initial version of the record provided with the data available from the Tel
+                    Aviv University Library <ptr
+                        target="https://tau.primo.exlibrisgroup.com/permalink/972TAU_INST/bai57q/alma9932995986004146"
+                    /></p>
+            </editionStmt>
+            <publicationStmt>
+                <authority>Hiob-Ludolf-Zentrum für Äthiopistik</authority>
+                <publisher>Die Schriftkultur des christlichen Äthiopiens und Eritreas: Eine
+                    multimediale Forschungsumgebung / Beta maṣāḥǝft</publisher>
+                <availability>
+                    <licence target="http://creativecommons.org/licenses/by-sa/4.0/">This file is
+                        licensed under the Creative Commons Attribution-ShareAlike 4.0. </licence>
+                </availability>
+            </publicationStmt>
+            <sourceDesc>
+                <msDesc xml:id="ms">
+                    <msIdentifier>
+                        <repository ref="INS0866TSL"/>
+                        <idno>Faitlovich 6</idno>
+                    </msIdentifier>
+                    <msContents>
+                        <summary>Fol. 1a. The book of the disciples (Arde'et). This is the Falasha version of the book Arde'et published by E. Littmann in the Journal of the American Oriental Society, vol. XXV, pp. 1-48, which represents the Christian version. The Falasha version has been published by Aescoly in his book Recueil de textes Falachas, Paris 1951, together with a French translation. His text is based on the manuscript of the collection d'Abbadie in the Bibliothèque Nationale of Paris. The text of the present manuscript shows numerous variant readings and complete certain lacunae in the text published by Aescoly. It includes the book designed as "Acts of Moses" (compare Wrights catalogue Ms CLXIX) and reaches until page 142 of Aescoly's printed text
```

Comment:
It is better to use "Beta ʾƎsrāʾel" instead of the term "falasha", except in titles.

Reply:
OK, but be aware: these stubs are not our descriptions, but imported from the library site as is. They are not corrected or revised.

Reply:
I already assumed, that it came from external data.